test/test_core.py

```diff
@@ -9119,8 +9119,13 @@ def test_pthread_create(self):
     self.do_run_in_out_file_test('core/pthread/create.cpp')
 
   @node_pthreads
-  def test_pthread_c11_threads(self):
-    self.set_setting('PROXY_TO_PTHREAD')
+  @parameterized({
+    'unpooled': ([],),
```

Comment:
Can you just use the empty string rather than `unpooled` here and below, which would just act as "default"?

Then the default version of the test has a name that matches the name of the functions, which is property I quite like.

Reply:
I don't feel super strongly either, but it makes sense to be for the default one to be the one with no extra args (or a default set of args, e.g. pool size of 0).